src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNetMvcIntegration.cs

```diff
@@ -74,12 +75,28 @@ public static Scope CreateScope(dynamic controllerContext)
                         // get route and routeValues from the RouteData in routeMatches
                         route = routeMatches[0].Route as Route;
                         routeValues = routeMatches[0].Values;
+
+                        if (route != null)
+                        {
+                            resourceName = $"{httpMethod} {route.Url.ToLowerInvariant()}";
```

Comment:
We should try to replace `{controller}` and `{action}` with they values. Otherwise, in some apps everything will have a resource name with something like `{controller}/{action}/{id}`, which is useless. You'll need to the the controller and action names earlier, since it's down in lines 92-93 right now.